src/commands/registries/azure/tasks/buildImageInAzure.ts

```diff
@@ -5,12 +5,24 @@
 
 import * as vscode from 'vscode';
 import { IActionContext, UserCancelledError } from '@microsoft/vscode-azext-utils';
+import { KnownRunStatus, Run } from '@azure/arm-containerregistry';
 import { scheduleRunRequest } from './scheduleRunRequest';
+import { sleep } from '../../../../utils/sleep';
 
-export async function buildImageInAzure(context: IActionContext, uri?: vscode.Uri | undefined): Promise<void> {
+const WAIT_MS = 500;
```

Comment:
I swapped to 5 seconds. IMO that seems like a good mix of not too fast but not too slow where a user is waiting and wondering what's happening (because they see in the logs that the run was completed).